packages/edit-site/src/components/sidebar/index.js

```diff
@@ -5,10 +5,16 @@ import { createSlotFill } from '@wordpress/components';
 import { ComplementaryArea } from '@wordpress/interface';
 import { __ } from '@wordpress/i18n';
 import { cog, pencil } from '@wordpress/icons';
+import { Platform } from '@wordpress/element';
 
 const { Slot: InspectorSlot, Fill: InspectorFill } = createSlotFill(
 	'EditSiteSidebarInspector'
 );
+
+const BLOCK_INSPECTOR_ACTIVE_BY_DEFAULT = Platform.select( {
```

Comment:
Is this default value different  between native and web or between web and web mobile?

Reply:
The difference is between native and web. We had a /packages/interface/src/store/defaults.native.js file specifying the default for native is to contain no open sidebars.